usaspending_api/broker/management/commands/load_fpds_from_broker.py

```diff
@@ -0,0 +1,135 @@
+from django.core.management.base import BaseCommand
+import logging
+import re
+import psycopg2
+from datetime import datetime
+
+from usaspending_api.data_load.fpds_loader import run_fpds_load, destory_orphans
+from usaspending_api.common.retrieve_file_from_uri import RetrieveFileFromUri
+from usaspending_api.common.helpers.date_helper import datetime_command_line_argument_type
+from usaspending_api.common.helpers.sql_helpers import get_broker_dsn_string
+from usaspending_api.common.helpers.etl_helpers import update_c_to_d_linkages
+from usaspending_api.etl.award_helpers import update_awards, update_contract_awards, update_award_categories
+from usaspending_api.broker.helpers.last_load_date import get_last_load_date, update_last_load_date
+
+logger = logging.getLogger("console")
+
+BROKER_CONNECTION_STRING = get_broker_dsn_string()
+
+CHUNK_SIZE = 10000  # Completely arbitrary and not backed by any testing, this can likely go higher
+
+ALL_FPDS_QUERY = "SELECT detached_award_procurement_id FROM detached_award_procurement;"
+
+
+class Command(BaseCommand):
+    help = "Sync USAspending DB FPDS data using Broker for new or modified records and S3 for deleted IDs"
+
+    modified_award_ids = []
+
+    @staticmethod
+    def get_cursor_for_date_query(date):
+        with psycopg2.connect(dsn=BROKER_CONNECTION_STRING) as connection:
+            db_cursor = connection.cursor()
+            db_query = ALL_FPDS_QUERY
+            if date:
+                db_query.format(" WHERE updated_at >= %s;")
+                db_args = [date]
+                db_cursor.execute(db_query, db_args)
+            else:
+                db_cursor.execute(db_query)
+        return db_cursor
+
+    def load_fpds_from_date(self, date):
+        if date is None:
+            logger.info("fetching all fpds transactions...")
+        else:
+            logger.info("fetching fpds transactions since {}...".format(str(date)))
+        cursor = self.get_cursor_for_date_query(date)
+        while True:
+            id_list = [id[0] for id in cursor.fetchmany(CHUNK_SIZE)]
```

Comment:
`fetchmany` without a named cursor fetches all of the results locally potentially defeating the intent of using `fetchmany` in the first place.

http://initd.org/psycopg/docs/usage.html#server-side-cursors